src/main/java/frc/robot/Constants.java

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot;
+
+import com.pathplanner.lib.util.PIDConstants;
+
+import edu.wpi.first.math.geometry.Translation3d;
+import edu.wpi.first.math.util.Units;
+import swervelib.math.Matter;
+import swervelib.parser.PIDFConfig;
+
+/**
+ * The Constants class provides a convenient place for teams to hold robot-wide
+ * numerical or boolean constants. This
+ * class should not be used for any other purpose. All constants should be
+ * declared globally (i.e. public static). Do
+ * not put anything functional in this class.
+ *
+ * <p>
+ * It is advised to statically import this class (or one of its inner classes)
+ * wherever the
+ * constants are needed, to reduce verbosity.
+ */
+public final class Constants {
+
+  public static final double ROBOT_MASS = (65) * 0.453592; // 32lbs * kg per pound
+  public static final Matter CHASSIS = new Matter(new Translation3d(0, 0, Units.inchesToMeters(8)), ROBOT_MASS);
+  public static final double LOOP_TIME = 0.13; // s, 20ms + 110ms sprk max velocity lag
```

Comment:
Have you considered putting the units in the name, e.g.:
 - public static final double ROBOT_MASS_KG
 - public static final double LOOP_TIME_S
 - public static final double MAX_ACCELERATION_M_S2
 - public static final double WHEEL_LOCK_TIME_S

This can help avoid people wondering what the unit is or having to ctrl-click to see the comment.

Reply:
I think the comments are fine for now, these get passed directly to yagsl, and aren’t utilized in our code.